glean-core/src/traits/boolean.rs

```diff
@@ -0,0 +1,23 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+/// A description for the `BooleanMetric` type.
+///
+/// When changing this trait, make sure all the operations are
+/// implemented in the related type in `../metrics/`.
+pub trait Boolean {
+    /// Sets to the specified boolean value.
+    ///
+    /// # Arguments
+    ///
+    /// * `value` - the value to set.
+    fn set(&self, value: bool);
+
+    /// **Test-only API (exported for FFI purposes).**
```

Comment:
The FFI part is not true anymore. This is becoming a general "exported for test purposes" part.

Reply:
(applies to all traits)